src/test/java/io/confluent/examples/streams/microservices/EndToEndTest.java

```diff
@@ -61,11 +63,12 @@ public void shouldCreateNewOrderAndGetBackValidatedOrder() throws Exception {
     assertThat(returnedBean.getState()).isEqualTo(OrderState.VALIDATED);
   }
 
-  @Test
+
+    @Test
   public void shouldProcessManyValidOrdersEndToEnd() throws Exception {
-    Client client = ClientBuilder.newClient();
+        final Client client = getClient();
```

Comment:
nit: fit indention

This is all over the place -- guess you nee to change you IDE setting -- note, examples and AK uses different code style

Reply:
i'll pick up fixing this in a follow up pr as I have the IDE settings configured.